x-pack/plugins/exploratory_view/public/components/shared/exploratory_view/embeddable/index.tsx

```diff
@@ -70,7 +70,7 @@ export function getExploratoryViewEmbeddable(
 
     const series = attributes[0];
 
-    const isDarkMode = uiSettings?.get('theme:darkMode');
+    const isDarkMode = theme?.getTheme().darkMode ?? false;
```

Comment:
I see this code is repeated across many places, can't we export a new hook in the `@kbn/kibana-react-plugin/public` that encapsulates this logic and always returns a boolean, would make things simpler for clients?

Reply:
Yeah, I guess while I'm at it, it takes sense.

Reply:
done in https://github.com/elastic/kibana/pull/173639/commits/9548c4a43b5cc78731ea0d816ed851d8fbd2eee9. I only adapted the usages that were already using a hook, not the usages retrieving the theme from core contracts directly.